doc/classes/EditorSettings.xml

```diff
@@ -1103,6 +1202,15 @@
 		<member name="text_editor/completion/use_single_quotes" type="bool" setter="" getter="">
 			If [code]true[/code], performs string autocompletion with single quotes. If [code]false[/code], performs string autocompletion with double quotes (which matches the [url=$DOCS_URL/tutorials/scripting/gdscript/gdscript_styleguide.html]GDScript style guide[/url]).
 		</member>
+		<member name="text_editor/external/exec_flags" type="String" setter="" getter="">
+			The command-line arguments to pass to the external text editor that is run when [member text_editor/external/use_external_editor] is [code]true[/code]. See also [member text_editor/external/exec_path].
+		</member>
+		<member name="text_editor/external/exec_path" type="String" setter="" getter="">
+			The path to the text editor executable used to edit text files if [member text_editor/external/use_external_editor] is [code]true[/code].
+		</member>
+		<member name="text_editor/external/use_external_editor" type="bool" setter="" getter="">
+			If [code]true[/code] opens an external editor instead of the built-in script editor. See also [member text_editor/external/exec_path] and [member text_editor/external/exec_flags].
```

Comment:
Depending on the desired behavior from:
* https://github.com/godotengine/godot/issues/93415

This might need clarification